packages/application-generic/src/usecases/get-feature-flag/get-is-request-rate-limiting-enabled.use-case.ts

```diff
@@ -1,21 +1,22 @@
 import { Injectable } from '@nestjs/common';
-import { SystemCriticalFlagsEnum } from '@novu/shared';
+import { FeatureFlagsKeysEnum } from '@novu/shared';
 
-import { GetSystemCriticalFlag } from './get-system-critical-flag.use-case';
+import { FeatureFlagCommand } from './get-feature-flag.command';
+import { GetFeatureFlag } from './get-feature-flag.use-case';
 
 @Injectable()
-export class GetIsRequestRateLimitingEnabled extends GetSystemCriticalFlag {
-  execute(): boolean {
-    const value = process.env.IS_REQUEST_RATE_LIMITING_ENABLED;
+export class GetIsApiRateLimitingEnabled extends GetFeatureFlag {
```

Comment:
Changing `GetFeatureFlag` -> `GetSystemCriticalFlag` and modifying the `execute` method accordingly.